home.html

```diff
@@ -0,0 +1,139 @@
+        <div class="footer">
+        <p class="date">2023.07.19(THU) <p>
+           <p class="address"> @ Macroyan, Kabul, Afghanistan</p>
+        </div>
+    </div>
+    </main>
+
+    <h2 class="background-img2"></h2>
+    <div class="card-imgs">
+     <div class="main-sports">    
+    <h1 class="main-sport">Main sports</h1>
+    <img class="bottom-line" src="images/Underline.png" alt="underline image">
+    </div>
+    <div class="cards"> 
+    <div class="card">
+    <img class="card-img-top" src="images/basketball.jpg" alt="Card image cap">
+    <div class="sports-name">
+        <h2 class="sport-name name-1">basketball</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-1">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+   
+    </div>
+    </div>
+    <div class="card">
+    <img class="card-img-top" src="images/Football.jpg" alt="Card image cap">
+    <div class="sports-name">
+        <h2 class="sport-name name-2">Football</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-2">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+   
+    </div>
+</div>
+    <div class="card">
+    <img class="card-img-top" src="images/cricket.jpg" alt="Card image cap">
+    <div class="sports-name">
+    <h2 class="sport-name name-3">Cricket</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-3">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    
+    </div>
+</div>
+    <div class="card">
+    <img class="card-img-top" src="images/Weighting.jpg" alt="Weigth liftting">
+    <div class="sports-name name-4">
+        <h2 class="sport-name">Weight lefting</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-4">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    </div>
+    </div>
+</div>
+    <button class="button1" href="./" type="submit">Join Sports Galla 2022</button>
+    <a class="button2" href="./" type="submit">SEE THE WHOLE PROGRAM</a>
+    </div>
+    </div>
+   
+ 
+    <section class="background-img3" id="background-img3">
+
+
+    </section>
+ 
+
+   <div class="background-img4">
+    <h2 class="background-img4th"></h2>
+     
+        <div class="partners">    
+        <h1 class="partner">Partners</h1>
+        <img class="bottom-line3" src="images/Underline.png" alt="underline image">
+        </div>
+        <div class="partner-logos">       
+        <img class="partner-logo" src="images/AGC logo.jpg" alt="Financial-supporters"> 
+        <img class="partner-logo" src="images/AWCC.jpg" alt="Financial-supporters">  
+        <img class="partner-logo" src="images/ATN.jpg" alt="Financial-supporter"> 
+        <img class="partner-logo" src="images/IoGlobal.png" alt="Financial-supporter">  
+        </div>
+  
+</div>
+ 
```

Comment:
- [ ] Please reduce the spacing between the partners' logo here in this section as it appears on the design given. Take a look at the following comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/205446895-9032c444-b5b6-4c4d-9352-0bf4a110d631.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/205446916-664de34d-118b-4b36-b926-60723e0620e5.png)